docs/content/docs/contributor-docs/release.md

```diff
@@ -61,15 +61,18 @@ Kubernetes Deployment; the Deployment's Pod image will refer to the exact
 Docker image tag matching the release tag.
 
 ## Release steps
-<// mkdocs does not support numbered lists with code blocks. So use 1) instead of 1. in numbered list. >
-1) First check out a git branch for your release:
+
+0) Rebase $SERVICE and Code-Generator repos with latest code
+
+1) Navigate to $SERVICE repo and check out a git branch for your release:
 ```bash
 export RELEASE_VERSION=v0.0.1
 git checkout -b release-$RELEASE_VERSION
```

Comment:
nit: should this be `git checkout -b release-$RELEASE_VERSION upstream/main` so that latest code is checked out.